src/components/api-reference/scoring.js

```diff
@@ -1,28 +1,1004 @@
+      <value>number</value>
+      <parts>array(<a href='#score-part-resource'>Score Part</a>)</parts>
+      <metadata>object(<a href='#score-resource-meta'>Score Metadata</a>)</metadata>
+      <warnings>array(string)</warnings>
+      <errors>array(string)</errors>
+    </data>`
+  },
+  fields: [
+    {
+      name: 'name',
+      value: 'string',
+      description: 'Name of score object',
+      notes: 'static'
+    },
+    {
+      name: 'title',
+      value: 'string',
+      description: 'Title of score object',
+      notes: 'static'
+    },
+    {
+      name: 'detail',
+      value: 'string',
+      description: 'Semantic version of scoring engine',
+      notes: 'Semantic version of the Scoring Engine used to compute the score Object'
+    },
+    {
+      name: 'value',
+      value: 'number',
+      description: 'Total score of submission',
+      notes: 'Final, total score for the submission'
+    },
+    {
+      name: 'parts',
+      value: 'array',
+      description: 'Array ofScoreParts',
+      notes: 'Performance category score parts, that contribute to total score'
+    },
+    {
+      name: 'metadata',
+      value: 'object',
+      description: 'Top level scoring metadata',
+      notes: ''
+    },
+    {
+      name: 'warnings',
+      value: 'array',
+      description: 'Array of Warnings',
+      notes: 'Scoring issues that do not halt the scoring process'
+    },
+    {
+      name: 'errors',
+      value: 'array',
+      description: 'Array of Errors',
+      notes: 'Scoring issues that halt the scoring process'
+    }
+  ],
+  metadata_messages: {
+    base: {
+      metadata: [
+        {
+          name: 'messages',
+          value: 'object',
+          description: 'Key identifiers with value strings as messages',
+          notes: ''
+        },
+        {
+          name: 'maxMediumContributionIA',
+          value: 'number',
+          description: 'Medium score contribution of IA measurements',
+          notes: 'Dependent on provider profile'
+        },
+        {
+          name: 'maxHighContributionIA',
+          value: 'number',
+          description: 'High score contribution for IA measurements',
+          notes: 'Dependent on provider profile'
+        },
+        {
+          name: 'maxContributionIA',
+          value: 'number',
+          description: 'Maximum score the IA category can contribute to the final score',
+          notes: 'Dependent on provider profile'
+        },
+        {
+          name: 'maxContributionACI',
+          value: 'number',
+          description: 'Maximum score the ACI category can contribute to the final score',
+          notes: 'Dependent on provider profile'
+        },
+        {
+          name: 'maxContributionQuality',
+          value: 'number',
+          description: 'Maximum score the Quality category can contribute to the final score',
+          notes: 'Dependent on provider profile'
+        }
+      ],
+      messages: [
```

Comment:
@sietekk this could be slightly confusing, maybe we must remove this section as it is only specific to UI display. OR if we want to retain this section, we must identify all keys that are possible in a message and explain them.
```
"metadata": {
                  "messages": {
                    "denominator": "At least 1 high or outcome or patient experience measure available. So denominator is 60",
                    "totalMeasurementPoints": "Sum of applicable individual measurement points.",
                    "totalBonusPoints": "Sum of individual measurement bonus points"
                  },
                  .....
}
```

Reply:
We do identify all possible keys, and the `description` property explains what the message key is for. The actual messages are dynamic in nature, so we can't simply cut/paste the messages here.